### PR TITLE
hw/ipc_nrf5340: Some updates

### PR DESCRIPTION
**Pull request by andrzej-kaczmarek**

### CODE REVIEW

File: hw/drivers/ipc_nrf5340/src/ipc_nrf5340.c [hunk 1]
```diff
@@ -470,6 +470,25 @@ ipc_nrf5340_available_buf(int channel, void **dptr)
     return 0;
 }
 
+uint16_t
+ipc_nrf5340_data_available_get(int channel)
```
Comment (sjanc) [inline hw/drivers/ipc_nrf5340/src/ipc_nrf5340.c:471]: there is already ipc_nrf5340_available() for this

File: hw/drivers/ipc_nrf5340/include/ipc_nrf5340/ipc_nrf5340.h [hunk 2]
```diff
@@ -131,6 +131,32 @@ uint16_t ipc_nrf5340_available(int channel);
  */
 uint16_t ipc_nrf5340_available_buf(int channel, void **dptr);
 
+/**
+ * Returns number of free space available in IPC ring buffer.
```
Comment (sjanc) [inline hw/drivers/ipc_nrf5340/include/ipc_nrf5340/ipc_nrf5340.h:151]: copy-paste?  same as ipc_nrf5340_data_available_get() and doesn't match implementation

File: hw/drivers/ipc_nrf5340/src/ipc_nrf5340.c (hunk 1 above)
Comment (andrzej-kaczmarek) [reply hw/drivers/ipc_nrf5340/src/ipc_nrf5340.c:471]: see the last commit in this PR

Comment (sjanc) [reply hw/drivers/ipc_nrf5340/src/ipc_nrf5340.c:471]: hmm fair enough,  but it could use ipc_nrf5340_shm_get_data_length() internally anyway ;)

Comment (andrzej-kaczmarek) [reply hw/drivers/ipc_nrf5340/src/ipc_nrf5340.c:471]: I'll think about it ;)

**Review by sjanc — APPROVED** [review]

**Pull request merged**